<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>654</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>372</width>
    <height>592</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>372</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hesap Makinesi</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/icon Hesap Makinesi.ico</normaloff>../images/icon Hesap Makinesi.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    border: 1px solid gray;
    border-radius: 5px;
    padding: 0 8px;
    background: white;
    selection-background-color: red;
}

 QToolButton{
	border: 2px solid gray;
	border-radius: 5px;
	font-size: 20px;
	padding: 10px;
	width: 20px;
	height:20px;
	background: gray;
	color:white;
}

QToolButton:hover{	
	 background: white;
	color:black;
}

QTextEdit{
	border: 1px solid gray;
	border-radius: 5px;
	background: #f2f2f2;
}

QToolButton#btnParantez,
QToolButton#btnYuzde
{
	 background: lightgray;
	border: 2px solid lightgray;
}

QToolButton#btnMemoryTemizle,
QToolButton#btnMemoryEkleArti,
QToolButton#btnMemoryEkleEksi,
QToolButton#btnMemoryCagir
{
	background: #64a2ff;
	border-color: #64a2ff;
	font-size: 16px;
}</string>
  </property>
  <widget class="QLineEdit" name="txtScreen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btnTemizle">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>520</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#f40352;
border: 2px solid #f40352;</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnParantez">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>()</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnYuzde">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>520</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnBolme">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>รท</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCarpma">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ร</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCikartma">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnToplama">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnEnter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#f40352;
background-color:white;
padding-top:5px;</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnVirgul">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;
font-weight:bold;
line-height: 12px;
padding-top:0px;
vertical-align:top</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn0">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtMainScreen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnMemoryCagir">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>MR</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnMemoryEkleEksi">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>M -</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnMemoryTemizle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>MC</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnMemoryEkleArti">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>M +</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="lblBellek">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>M : </string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="btnArtiEksi">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
